core/src/com/projectkorra/projectkorra/waterbending/multiabilities/WaterArms.java

```diff
@@ -757,4 +758,17 @@ public void addToArm(final Block block, final Arm arm) {
 			this.right.add(block);
 		}
 	}
+
+	public void emitLight(final Location location) {
+		if (!getConfig().getBoolean("Properties.Fire.DynamicLight.Enabled")) return;
+
+		int brightness = getConfig().getInt("Properties.Fire.DynamicLight.Brightness");
+		long keepAlive = getConfig().getLong("Properties.Fire.DynamicLight.KeepAlive");
+
+		if (brightness < 1 || brightness > 15) {
+			throw new IllegalArgumentException("Properties.Fire.DynamicLight.Brightness must be between 1 and 15.");
+		}
+
+		LightManager.createLight(location).brightness(brightness).timeUntilFadeout(keepAlive).emit();
+	}
```

Comment:
Is there a reason this exists and we don't use `emitFirebendingLight` instead?

Reply:
WaterArms doesn't extend FireAbility and has no access to the class method `emitFirebendingLight`

Reply:
> WaterArms doesn't extend FireAbility and has no access to the class method `emitFirebendingLight`

Should the function perhaps be a static one?

Reply:
I was informed there is an interest in keeping static and configuration related things separate. Makes sense to me.